README.md

```diff
@@ -189,9 +189,8 @@ Example:
 
```

Comment:
![image](https://github.com/unorjikingsley/Vet-Clinic-Database/assets/93039359/4347f174-d85f-4620-af6d-020b73a18091)

- [ ] It would be nice if fix the links highlighted in the image above because right now it is broken 😨 Kindly fix them or remove them (If you don't have any particular section for these links) to make your readme descriptive and professional 👍